engine/api/src/main/scala/pl/touk/nussknacker/engine/api/typed/MapLikePropertyTypeDeterminer.scala

```diff
@@ -0,0 +1,28 @@
+package pl.touk.nussknacker.engine.api.typed
+
+import java.util
+
+import pl.touk.nussknacker.engine.api.typed.typing.{Typed, TypedClass, TypingResult, Unknown}
+
+object MapLikePropertyTypeDeterminer {
```

Comment:
I wonder, maybe it should be in spel package? It's currently used only in Typer as far as I see, and it's behaviour is heavily dependent on property accessors in spel?
We would be able to use Avro classes then?

Reply:
Good point - moved to typer, it shouldn't cause problems with Avro classes